includes/admin/llms.functions.admin.php

```diff
@@ -288,3 +288,33 @@ function llms_merge_code_button( $target = 'content', $echo = true, $codes = arr
 	return $html;
 
 }
+
+/**
+ * Retrieve the precision for round function for floating values.
+ *
+ * The function returns precision to round off the decimal values
+ * for 'average' column in reporting tables.
```

Comment:
I missed this, can we have a more generic description here? We're gonna use it in several places.
Or even remove it.
Thanks

Reply:
Removed 👍